app/models/concerns/generic_phrase_concern.rb

```diff
@@ -2,12 +2,14 @@ module GenericPhraseConcern
   extend ActiveSupport::Concern
 
   class_methods do
-    def co_occurring_generic_phrases(generic_phrases_for_survey_answers)
+    def co_occurring_generic_phrases(generic_phrases_for_survey_answers, generic_phrase)
       generic_phrase_combinations = []
 
       generic_phrases_for_survey_answers.group_by(&:first).each do |_, survey_answer_id_generic_phrases_pair|
```

Comment:
I'm being picky but could this be an `each_with_object`?

Reply:
I've tested this and it's failing on the tests without being immediately obvious why. Maybe a refactoring down the line, but for now I'm going to keep as-is.

Reply:
Fair enough, minor thing